curriculum/challenges/english/25-front-end-development/lab-customer-complaint-form/67279fe50237291f80eed8b8.md

```diff
@@ -175,24 +175,26 @@ assert.isFalse(validateForm()["complaints-group"]);
 Once one checkbox from `#complaints-group` is checked, you should set `#complaints-group`'s border color to `green`.
 
 ```js
-document.getElementById("damaged-product").checked = true;
+const damagedProduct = document.getElementById("damaged-product");
+damagedProduct.checked = true;
 document.getElementById("nonconforming-product").checked = false;
 document.getElementById("delayed-dispatch").checked = false;
 document.getElementById("other-complaint").checked = false;
 const fieldset = document.getElementById("complaints-group");
-fieldset.dispatchEvent(new Event("change", { bubbles: true }));
+damagedProduct.dispatchEvent(new Event("change", { bubbles: true }));
```

Comment:
Could we swap these lines (and in other added cases)? To move `fieldset` closer to where it's used in assertion.